draft-ounsworth-cfrg-kem-combiners.mkd

```diff
@@ -228,40 +251,55 @@ ss = kemCombiner(ss_1, ss_2, ..., ss_n)
 
 This document assumes that shared secrets are the output of a KEM, but without loss of generality they MAY also be any other source of cryptographic key material, such as pre-shared keys (PSKs), with PQ/PSK being a quantum-safe migration strategy being made available by some protocols, see for example IKEv2 in [RFC8784].
 
-In general it is desirable to use a multi-PRF as a KEM combiner, meaning a PRF when keyed by any of its single inputs.
+In general it is desirable to use a multi-PRF as a KEM combiner, meaning that the combiner has the properties of a PRF when keyed by any of its single inputs.
 The following simple yet generic construction can be used in all IETF protocols that need to combine the output of two or more KEMs:
 
 ~~~
 KDF(counter || k_1 || ... || k_n || fixedInfo, outputBits)
 ~~~
-{: title="general KEM combiner construction" }
+{: #tab-kemCombiner title="general KEM combiner construction" }
 
 where:
 
 - `KDF` represents a suitable choice of cryptographic key derivation function,
-- `k_i` represent the constant-length input keys,
+- `k_i` represent the constant-length input keys and is discussed in {{sec-k_i-construction}},
 - `fixedInfo` is some protocol-specific KDF binding,
 - `counter` parameter is instantiation-specific and is discussed in {{sec-instantiation}}.
-- `outputBits` determines the length of the key,
+- `outputBits` determines the length of the key to output,
 - `||` represents concatenation.
 
-In section {{sec-instantiation}} are listed several possible practical instantiations, in compliance with NIST SP-800 56Cr2 {{SP800-56C}}.
-The shared secret `ss` MAY be used as a Key Encryption Key (KEK) for key encapsulation.
+In {{sec-instantiation}} are listed several possible practical instantiations, in compliance with NIST SP-800 56Cr2 {{SP800-56C}}.
```

Comment:
```suggestion
In {{sec-instantiation}} several possible practical instantiations are listed, that are in compliance with NIST SP-800 56Cr2 {{SP800-56C}}.
```
Felt that "are listed" had to be put to the end, but maybe my english is too bad.

Just a remark here: The inclusion of SHAKE is not compliant to NIST SP-800 56Cr2. They only list SHA3 and KMAC. But I would say leave the sentence as it is here. I'll take up the discussion in the section where SHAKE is listed.

Reply:
Added manually to my branch.